backend/routes/events.js

```diff
@@ -29,19 +29,19 @@ router.get("/:id", function (req, res, next) {
 });
 
 router.post("/", function (req, res, next) {
-  let startDate = new Date(req.body.start + " UTC");
-  let endDate = new Date(req.body.end + " UTC");
+  let startDate = new Date(req.body.start);
+  let endDate = new Date(req.body.end);
```

Comment:
Since you dump most of req.body into newEvent, maybe you can just new Event({...req.body, participantSize: "1"}), etc.

Reply:
I don't quite understand what you mean